content/how-to/dedicated-hosting/how-to-request-a-csr/index.md

```diff
@@ -1,41 +1,43 @@
 ---
 permalink: how-to-request-a-csr
 audit_date:
-title: 'How to request a CSR'
+title: 'How to Request a CSR'
 type: article
 created_date: '2021-04-15'
 created_by: Alexandra Soares
-last_modified_date: '2021-07-07'
-last_modified_by: Cat Lookabaugh
+last_modified_date: '2023-02-01'
+last_modified_by: Asmita Nakwa
 product: Dedicated Hosting
 product_url: dedicated-hosting
 ---
 
 This article explains how to create a ticket to generate a Certificate Signing
 Request (CSR).
 
-### Create a ticket in the MyRackspace portal
+### Create a ticket in the MyRackspace Portal
 
-1. Log in to [MyRackspace Portal](https://login.rackspace.com/login) with your username and password.
+**Step 1.** Log in to [MyRackspace Portal](https://login.rackspace.com/login) with your username and password.
 
-2. On the top navigation bar, choose **Select a Product** > **Rackspace Dedicated**.
+   <img width="409" alt="Enter the login credentials" src="/support/how-to/how-to-request-a-csr/csr1.png">
 
-3. Select **Tickets** > **Create Ticket**. The **Create New Ticket** page displays.
+**Step 2.** On the top navigation bar, select **Support** > **Support Center**.
 
-4. Select the **Subject** field.
+   <img width="941" alt="Select the supportcenter" src="/support/how-to/how-to-request-a-csr/csr2.png">
 
-5. From the drop-down menu, select **Generate Certificate Signing Request (CSR)**.
+**Step 3.** Under **Common Request** menu, click **See All Templates**. The **See All Templates** displays list of all templates. Select **Generate Certificate Signing Request (CSR)**
 
-   {{<image src="CSR1.png" alt="" title="">}}
+   <img width="960" alt="Select CSR from the list" src="/support/how-to/how-to-request-a-csr/csr4.png">
+   
+**Step 4.** In **Issue Details** section you can select a device from the drop-down menu.
 
-6. In the **Ticket Details** section, select a device from the drop-down menu.
+   <img width="569" alt="Selecr devices from the drop down list" src="/support/how-to/how-to-request-a-csr/csr5.png">
 
-7. Complete the following form:
+**Step 5.** Complete the following form:
 
-   {{<image src="CSR2.png" alt="" title="">}}
+   <img width="391" alt="Signing request form" src="/support/how-to/how-to-request-a-csr/csr6.png">
```

Comment:
Is the device in this image using production data or did we alter it already?

Reply:
Yes, the image is from real production data. I have replaced the image since it is showing device information.